torchvision/ops/boxes.py

```diff
@@ -133,6 +133,78 @@ def clip_boxes_to_image(boxes: Tensor, size: Tuple[int, int]) -> Tensor:
     return clipped_boxes.reshape(boxes.shape)
 
 
+def box_cxcywh_to_xyxy(boxes: Tensor) -> Tensor:
+    """
+    Converts bounding boxes from (cx, cy, w, h) format to (x1, y1, x2, y2) format.
+
+    Arguments:
+        boxes (Tensor[N, 4]): boxes in (cx, cy, w, h) format which will be converted.
+
+    Returns:
+        boxes (Tensor(N, 4)): boxes in (x1, y1, x2, y2) format.
+    """
+    # We need to change all 4 of them so some temporary variable is needed.
+    x1 = boxes[:, 0] - (0.5 * boxes[:, 2])
+    y1 = boxes[:, 1] - (0.5 * boxes[:, 3])
+    x2 = boxes[:, 0] + (0.5 * boxes[:, 2])
+    y2 = boxes[:, 1] + (0.5 * boxes[:, 3])
```

Comment:
For less typing an a bit clearer reading, I propose we unpack first to different coordinates:
```suggestion
    cx, cy, w, h = boxes.unbind(-1)
    x1 = cx - 0.5 * w
    y1 = cy - 0.5 * h
    x2 = cx + 0.5 * w
    y2 = cy + 0.5 * h
```